### ISSUE TITLE
Fitted Block Bounds are small and can cause lag during drag on slow computers

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/number-line-integers#77]
**Test device**
Lovelace
**Operating System**
ChromeOS
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/465.
During rc.1 I noted (https://github.com/phetsims/QA/issues/465#issuecomment-572638196):
>Lagging occurs in ChromeOS when dragging objects around. It is especially bad when outside of the play areas.

@jbphet noticed that this was due to the bounds that determine how objects are drawn with canvas. This occurs on any platform it seems, but it causes lag in slower computers like the chromeOS device. It does seem to happen on master.

**Visuals**
![newboundsissue](https://user-images.githubusercontent.com/41024075/72107088-e17e2500-32ed-11ea-953f-c51690e13a59.gif)
![performancefittedblock](https://user-images.githubusercontent.com/41024075/72107089-e17e2500-32ed-11ea-9754-35a1ef5d4aa1.png)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Number Line: Integers‬
URL: https://phet-dev.colorado.edu/html/number-line-integers/1.0.0-rc.1/phet/number-line-integers_en_phet.html?showFittedBlockBounds
Version: 1.0.0-rc.1 2020-01-06 19:43:39 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
Query Parameter used: ?showFittedBlockBounds

**Comment by jbphet** [comment]
I discussed this with @jonathanolson and he recommended adding the `preventFit` flag to the explore screen, so that's what I've done.  It seems to fix the reported behavior, but I don't know whether there are other consequences, so we should test this thoroughly on the next RC.

**Comment by KatieWoe** [comment]
Mostly fixed. On the generic screen this may still happen to an extent. Up to you @jbphet 
![stillboundsabit](https://user-images.githubusercontent.com/41024075/72845114-26457c80-3c5b-11ea-9278-132bac9513a6.gif)

**Comment by jbphet** [comment]
I tested this out and the performance change is not noticeable on my system.  Given that the impact is small, the portion of the screen where the resizing occurs is outside the normal range where these points will be placed and therefore unlikely to be a problem for users, and also that preventing the fitting of block bounds comes with an increase in memory usage, I'm going to leave this one as is.  Closing.
